### ISSUE TITLE
"Please check the log" -- where is the log?

### ISSUE THREAD

**Issue by tjluoma** [andreafabrizi/Dropbox-Uploader#517]
I just updated to the current version and created a new Dropbox 'app' and ~/.dropbox_uploader but nothing works except 'space' and 'info'. 

For example:

```
 dropbox_uploader.sh list '/Shared'
 > Listing "/Shared"... FAILED
Some error occured. Please check the log.
```

and also

```
 dropbox_uploader.sh upload 'Lessons and Carols (Readings).pdf' /Shared/
 > Uploading "Lessons and Carols (Readings).pdf" to "/Shared/Lessons and Carols (Readings).pdf"... FAILED
An error occurred requesting /upload
Some error occured. Please check the log.
```

I would be happy to check the log… but I don't know where to find the log.

**Comment by telemaxx** [comment]
have you seen issue #514?

**Comment by tjluoma** [comment]
I hadn't, but it led me to figure out that I had to go into another screen on the Dropbox.com website to add permissions, which really ought to be part of the instructions when creating a new ~/.dropbox_uploader file. Once I did that and re-ran the process with the app token, etc then I got it to work.

The location of the log is still entirely unclear. IMO, if there is no log unless you use `-d` then it shouldn't say "Check the log" it should say something like "Re-run with the `-d` and then look at '/tmp/du_resp_debug'."

**Comment by telemaxx** [comment]
yes,  that is right. and when starting from scratch the instructions are given from Dropboxuploader.sh should also be updated. when i find some time i send a PR

**Comment by telemaxx** [comment]
https://github.com/andreafabrizi/Dropbox-Uploader/pull/519
